README.md

```diff
@@ -1,13 +1,28 @@
 # Airalo Assessment
 
 ## Requirements
-    PHP >= 8.0
+- PHP >= 8.0
 
 ## Installation
-    cd [YOUR WORKSPACE]
-    mkdir -p airalo-assessment/project   
-    cd airalo-assessment/project
-    git clone git@github.com:Airalo/backend-assessment.git .
+```
+cd [YOUR WORKSPACE]
+mkdir -p airalo-assessment/project   
+cd airalo-assessment/project
+git clone git@github.com:Airalo/backend-assessment.git .
+```
 
 ### Composer dependencies
-    composer install
+```
+composer install
+```
+
+### Alternatively - Using Docker
+Ensure your `.env` file is set with `DB_*` connection details.
```

Comment:
It would be great if we add this detail.
`DB_HOST` must be `mariadb`

Reply:
I can add a `.env.docker` and add instructions to copy that file to `.env`